public/gameInfo.php

```diff
@@ -666,78 +671,74 @@ function ResetProgress() {
-                        echo "<form class='mb-3' action='/request/game/recalculate-points-ratio.php' method='post'>";
-                        echo csrf_field();
-                        echo "<input type='hidden' name='game' value='$gameID'>";
-                        echo "<button>Recalculate True Ratios</button>";
-                        echo "</form>";
-                    }
 
                     // Display leaderboard management options depending on the current number of leaderboards
-                    echo "<div class='mb-3'>";
+                    echo "<div>";
                     if ($numLeaderboards == 0) {
                         echo "<form action='/request/leaderboard/create.php' method='post'>";
                         echo csrf_field();
                         echo "<input type='hidden' name='game' value='$gameID'>";
                         echo "<button class='btn'>Create First Leaderboard</button>";
                         echo "</form>";
                     } else {
-                        echo "<div><a href='/leaderboardList.php?g=$gameID'>Manage Leaderboards</a></div>";
+                        echo "<div><a class='btn btn-link' href='/leaderboardList.php?g=$gameID'>Leaderboards</a></div>";
                     }
                     echo "</div>";
 
-                    echo "<div class='mb-1'><a href='/ticketmanager.php?g=$gameID'>View tickets for this game</a></div>";
-                    echo "<div class='mb-3'>";
+                    echo "<div><a class='btn btn-link' href='/codenotes.php?g=$gameID'>Code Notes</a></div>";
+
+                    if ($permissions >= Permissions::Developer) {
+                        echo "<div class='mb-3'><a class='btn btn-link' href='/managehashes.php?g=$gameID'>Hashes</a></div>";
+                    }
+
+                    echo "</div>";
+                    // right column
+                    echo "<div class='grow'>";
+
+                    if ($permissions >= Permissions::Developer) {
+                        echo "<form action='/request/game/recalculate-points-ratio.php' method='post'>";
+                        echo csrf_field();
+                        echo "<input type='hidden' name='game' value='$gameID'>";
+                        echo "<button>Recalculate True Ratios</button>";
+                        echo "</form>";
+                    }
+
+                    echo "<div><a class='btn btn-link' href='/ticketmanager.php?g=$gameID'>Tickets</a></div>";
+                    echo "<div>";
                     $isSubscribedToTickets = isUserSubscribedTo(SubscriptionSubjectType::GameTickets, $gameID, $userID);
                     RenderUpdateSubscriptionForm(
                         "updateticketssub",
                         SubscriptionSubjectType::GameTickets,
                         $gameID,
                         $isSubscribedToTickets,
-                        'tickets'
+                        'Tickets'
```

Comment:
There's also a "Subscribe to achievement comments" button below the rich presence. It should probably be brought up here.

Reply:
I think I put that down there on purpose to match other comments areas where the subscription button is on top. I can move it back though if you think it feels more natural to have it up there again

Reply:
Moved back up.